src/components/Game.jsx

```diff
@@ -7,17 +8,64 @@ class Game extends Component {
 
   constructor() {
     super()
+    this.board = shuffleBoard()
+    this.handleClick = this.handleClick.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.state = {
+      currentWord: '',
+      wordList: {},
+      totalScore : 0
+    }
   }
 
+  //tile() {
+  //    const tile = {
+  //      letter: ,
+  //      isSelected: false,
+  //      position: {row: rowIndex, column: col}
+  //    }
+  //}
+  handleClick(letter) {
+    this.setState({
+      currentWord: this.state.currentWord.concat(letter)
+    })
+  }
+
+  calculateScore(word) {
+    return word.length
```

Comment:
Wasn't there more logic to the scoring?  It was max of 8 or something and word.length - 2 I think.